sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -267,6 +299,37 @@ public void testUpdate() {
         assertEquals(person, updated);
     }
 
+    @Test
+    public void testPatch() {
+        insertedPerson = cosmosTemplate.patch(insertedPerson, operations, null);
+        Person patchedPerson = cosmosTemplate.findById(containerName, insertedPerson.getId(), Person.class);
+        assertEquals(insertedPerson.getAge(), PATCH_AGE_1);
```

Comment:
`patchedPerson` is not used

Reply:
No this is from the old tests before they were updated. @TheovanKraay please remove.

Reply:
Done.